### PR TITLE
Feature/absolute imports

### LINKED ISSUE #382: Use absolute imports
## Are you reporting a bug or a feature?

 - [ ] Bug
 - [x] Feature

## Expected Behavior

Can use absolute imports instead of relative '../../js/' etc.

## Actual Behavior

Relative imports

## Steps to Reproduce the Problem

Add a new component and have fun doing some relative path imports

### PR DESCRIPTION
**Pull request by TheDadi**
Fixes #382 .

Changes proposed in this pull request:

 - Use absolute imports

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
 
 # Checklist:
 
 - [x] My code follows the style guidelines of this project
 - [x] I have performed a self-review of my own code
 - [x] I have commented my code, particularly in hard-to-understand areas
 - [x] I have made corresponding changes to the documentation
 - [x] My changes generate no new warnings
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by AndyOGo** [comment]
really like it, we need similar for sass [`includePaths`](https://github.com/sass/node-sass#includepaths).

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,7737 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (AndyOGo) [inline yarn.lock]: @TheDadi 
Please remove the yarn.lock from git

Comment (AndyOGo) [reply yarn.lock]: and may add it to `.gitignore`

File: .gitignore [hunk 2]
```diff
@@ -39,3 +39,6 @@ jspm_packages
 
 
 package-lock.json
+
+
+.idea
```
Comment (LucaMele) [inline .gitignore]: developer IDE of choice stuff should not mess the repo. Please use global git ignore for that

Comment (AndyOGo) [reply .gitignore]: agree, gloabl git ignore config should be set.
Though we should also safeguard ourselves

Comment (LucaMele) [reply .gitignore]: Thats why we have to have PR approvals :) Developers IDE of choice stuff have nothing to do in a repo. We cannot add all the different existing IDE settings in the gitignore. We just have to have an Eye open on the PRs :)

**Comment by AndyOGo** [comment]
any news on this

**Comment by AndyOGo** [comment]
**Breaking Changes:** Please migrate before merging
https://github.com/axa-ch/patterns-library/releases/tag/v2.0.1-beta.217

**Pull request closed without merging**